index.html

```diff
@@ -87,7 +87,7 @@ <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
           </div>
         </div>
         <div class="card">
-          <img class="card-img" src="Images/SnapshootPortfolio.png" alt="" />
+          <img class="card-img" src="Images/SnapshootPortfolio.png" alt="This is project 2 image" />
           <div class="card-flex-container">
             <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
```

Comment:
- [ ] There is a contrast issue with the "See Project" button. I would suggest fixing this for better accessibility. I think you could try adding `font-weight` to the text to make it bold or changing a bit the hover color :wink:

  Button score:
  ![image](https://user-images.githubusercontent.com/81522435/139951299-27c28e53-65cf-4467-9101-2d44fb33b35c.png)

Reply:
Done